renovate.json

```diff
@@ -3,5 +3,36 @@
   "extends": [
     "config:best-practices",
     ":automergeStableNonMajor"
+  ],
+  "packageRules": [
+    {
+      "matchUpdateTypes": [
+        "patch",
+        "minor"
+      ],
+      "matchManagers": [
+        "pip_requirements"
```

Comment:
== dev-requirements.txt

Reply:
Is it worth switching to json5 so we can include these comments in the source?

Reply:
With the best-practices config we're now using, Renovate may convert it back to .json at any time, removing the comments. I tried to make the description sufficient that it's clear what's going on, I can be even more explicit?